.rubocop.yml

```diff
@@ -1,12 +1,28 @@
+AllCops:
+  TargetRubyVersion: 2.3
+  Exclude:
+    - 'bin/**/*'
+    - 'db/**/*'
+    - 'config/**/*'
+    - 'spec/**/*'
+
 Documentation:
   Enabled: false
 
+Rails:
+  Enabled: true
+
+# There is a bug in the Rubocop 0.48 parser that causes it to incorrectly
+# complain about parenthese around blocks used with scopes. This should be
+# removed once it is possible to upgrade to 0.49 or above.
+Lint/AmbiguousBlockAssociation:
+  Enabled: false
+
 Style/FrozenStringLiteralComment:
   Enabled: false
 
-AllCops:
-  Exclude:
-    - 'db/**/*'
-    - 'config/**/*'
-    - 'spec/**/*'
-    - !ruby/regexp /old_and_unused\.rb$/
+# Rubocop should be used for style guide adherence, not testing for code smell
+Rails/OutputSafety:
+  Enabled: false
+Rails/SkipsModelValidations:
+  Enabled: false
```

Comment:
Sorry to flip-flop on this issue, but maybe we want model validations? It seems a little odd to me to disable it everywhere because of that `update_all`. Should we just add validations to that? Thoughts?